### PR TITLE
[MIRROR] Some injection fixes

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/39642
--------------------
Fixes a breaking error and possible injection exploit  in `populate_key_2018-07-09.py` caused by a key existing from when byond allowed now blocked characters like `'` in usernames i.e http://www.byond.com/members/immortaljrosh?format=text\

Additionally fixes some unsanitized values in other queries.

### CODE REVIEW

**Comment by yoyobatty** [comment]
The diff changes decided to take a shit but it's identical from the original pr.

File: code/modules/admin/DB_ban/functions.dm [hunk 1]
```diff
@@ -1,550 +1,555 @@
...
-			if(adm_bans >= max_bans)
-				to_chat(usr, "<span class='danger'>You already logged [max_bans] admin ban(s) or more. Do not abuse this function!</span>")
-				qdel(query_check_adminban_amt)
-				return
-		qdel(query_check_adminban_amt)
-	if(!computerid)
-		computerid = "0"
-	if(!ip)
-		ip = "0.0.0.0"
-	var/internet_address_to_use = CONFIG_GET(string/internet_address_to_use)
```
Comment (JohnGinnane) [inline code/modules/admin/DB_ban/functions.dm:146]: I noticed this has also changed, did we miss something from tg? @Carbonhell

**Review by JohnGinnane — DISMISSED** [review]
Looks good other than `internet_address_to_use` being replaced with `world.internet_address` but I think we should have done that a while ago along with tg

File: code/modules/admin/DB_ban/functions.dm [hunk 2]
```diff
@@ -1,550 +1,555 @@
...
-	var/sql = "INSERT INTO [format_table_name("ban")] (`bantime`,`server_ip`,`server_port`,`round_id`,`bantype`,`reason`,`job`,`duration`,`expiration_time`,`ckey`,`computerid`,`ip`,`a_ckey`,`a_computerid`,`a_ip`,`who`,`adminwho`) VALUES (Now(), INET_ATON(IF('[internet_address_to_use]' LIKE '', '0', '[internet_address_to_use]')), '[world.port]', '[GLOB.round_id]', '[bantype_str]', '[reason]', '[job]', [(duration)?"[duration]":"0"], Now() + INTERVAL [(duration>0) ? duration : 0] MINUTE, '[ckey]', '[computerid]', INET_ATON('[ip]'), '[a_ckey]', '[a_computerid]', INET_ATON('[a_ip]'), '[who]', '[adminwho]')"
```
Comment (yoyobatty) [inline code/modules/admin/DB_ban/functions.dm:147]: I see the reference to the abovementioned variable

Comment (JohnGinnane) [reply code/modules/admin/DB_ban/functions.dm:147]: yeah but we'll be using world.internet_address so it should be grand

**Review by JohnGinnane — APPROVED** [review]
Alright should be good now

**Pull request merged**